### PR TITLE
0.16.1 release notes

### PR DESCRIPTION
**Pull request by ellismg**
@lukehoban I did the first cut, but I'll need you to fill in information about the resource providers.

All the packages have been published.

### CODE REVIEW

File: reference/changelog.md [hunk 1]
```diff
@@ -76,6 +81,42 @@ redirect_from: /install/changelog.html
     </tbody>
 </table>
 
+## v0.16.1 {#v161}https://github.com/pulumi/pulumi/pull/2069
+
+Released on October 15, 2018
```
Comment (ericrudder) [inline reference/changelog.md]: is this really the right date?

File: reference/changelog.md [hunk 2]
```diff
@@ -76,6 +81,42 @@ redirect_from: /install/changelog.html
...
+- Fixed an issue with the generated type for an Unwrap expression when using TypeScript [pulumi/pulumi#2061](https://github.com/pulumi/pulumi/pull/2061)
+- Improve error messages when resource plugins can't be loaded or when a checkpoint is invalid [pulumi/pulumi#2078](https://github.com/pulumi/pulumi/pull/2078)
+- Fix link to the Pulumi Web Console in the CLI for a stack [pulumi/pulumi#2075](https://github.com/pulumi/pulumi/pull/2075)
+- Attach git commit metadata to Pulumi updates in some additional cases [pulumi/pulumi#2062](https://github.com/pulumi/pulumi/pull/2062) and [pulumi/pulumi#2069](https://github.com/pulumi/pulumi/pull/2069)
+
+### @pulumi/aws 0.16.1
+
+- Update to the v1.14.0 version of the AWS Provider [pulumi/pulumi-aws#355](https://github.com/pulumi/pulumi-aws/pull/355)
+- Add definitions for S3 Canned ACLs [pulumi/pulumi-aws#351](https://github.com/pulumi/pulumi-aws/pull/355)
+- Add serverless methods to Kenisis streams [pulumi/pulumi-aws#345](https://github.com/pulumi/pulumi-aws/pull/345)
```
Comment (ericrudder) [inline reference/changelog.md]: Kinesis

File: reference/changelog.md [hunk 3]
```diff
@@ -76,6 +81,42 @@ redirect_from: /install/changelog.html
...
+
+### @pulumi/azure 0.16.3
+
+- Update to the v1.17.0 version of the Azure Provider [pulumi/pulumi-azure#137](https://github.com/pulumi/pulumi-azure/pull/137)
+- Fix an issue where the provider could fail with an error about a go type converison failing [pulumi/pulumi-azure#138](https://github.com/pulumi/pulumi-azure/pull/138)
```
Comment (ericrudder) [inline reference/changelog.md]: Go

File: reference/changelog.md [hunk 4]
```diff
@@ -117,7 +158,7 @@ In addition to the above features, we've made a handfull of day to day improveme
 
 ### @pulumi/aws 0.16.0
 
-- Moved all `@pulumi/aws-serverless` features down into `@pulumi/aws`.  Event handlers are now available as methods like `bucket.onObjectCreated` off of instances of `aws.s3.Bucket` and all other Lambda event sources.  In addition, the `aws.apigateway.x.API` type is available for simple construction of API Gateways directly in the `@pulumi/aws` package.  
+- Moved all `@pulumi/aws-serverless` features down into `@pulumi/aws`.  Event handlers are now available as methods like `bucket.onObjectCreated` off of instances of `aws.s3.Bucket` and all other Lambda event sources.  In addition, the `aws.apigateway.x.API` type is available for simple construction of API Gateways directly in the `@pulumi/aws` package.
```
Comment (ericrudder) [inline reference/changelog.md]: maybe "from" instead of "off of"

File: reference/changelog.md (hunk 1 above)
Comment (ellismg) [reply reference/changelog.md]: No, it is not.  Copy paste strikes again.

File: reference/changelog.md (hunk 2 above)
Comment (ellismg) [reply reference/changelog.md]: Thanks!

File: reference/changelog.md (hunk 3 above)
Comment (ellismg) [reply reference/changelog.md]: Sure.

File: reference/changelog.md (hunk 4 above)
Comment (ellismg) [reply reference/changelog.md]: Sure.

**Pull request merged**